javaslang-benchmark/src/test/java/javaslang/MemoryUsage.java

```diff
@@ -0,0 +1,36 @@
+package javaslang;
```

Comment:
Interesting. My hope is that the Bit-Mapped Vector Trie will make a difference for Vector.